todo_app/todo_card.py

```diff
@@ -1,10 +1,9 @@
 from datetime import datetime
 
-class TrelloCard:
+class TodoCard:
 
-    def __init__(self, list_name, list_id, card_id, card_name, last_modified): 
+    def __init__(self, list_name, card_id, card_name, last_modified): 
         self.status=list_name
-        self.list_id=list_id
         self.id=card_id
         self.title=card_name
         self.last_modified = datetime.strptime(last_modified, "%Y-%m-%dT%H:%M:%S.%fZ").date()
```

Comment:
Your code currently creates the "last_modified" string with with `item["date_last_activity"].strftime("%Y-%m-%dT%H:%M:%S.%fZ")`

And then this init function parses that and converts it back into a date.

In `get_cards_for_board` you should just pass in the datetime directly; don't use strftime. 
And then here you could just write:
```suggestion
        self.last_modified = last_modified.date()
```